### PR TITLE
Rewrite of chapter 2 for microbit v2

### PR DESCRIPTION
**Pull request by hargoniX**

### CODE REVIEW

File: src/02-requirements/README.md [hunk 1]
```diff
@@ -10,19 +10,22 @@ the fact that `extern crate` is not necessary in the 2018 edition.
 
 Also, to follow this material you'll need the following hardware:
 
-- A [micro:bit] board.
+- A [micro:bit v2] board, alternatively a [micro:bit v1] board.
```
Comment (eldruin) [inline src/02-requirements/README.md:13]: Should we say preferably V1.5? since V1.3 has some differing peripherals.

**Review by eldruin — COMMENTED** [review]
Looks good to me, thanks!

File: src/02-requirements/README.md (hunk 1 above)
Comment (hargoniX) [reply src/02-requirements/README.md:13]: Yeah being more precise with versions can never hurt

**Review by eldruin — APPROVED** [review]
Great, thank you!

bors r+

**Pull request merged**